druntime/mak/COPY

```diff
@@ -559,45 +559,30 @@ COPY=\
 	$(IMPDIR)\etc\valgrind\valgrind.d \
 	$(IMPDIR)\etc\linux\memoryerror.d \
 	\
-	$(IMPDIR)\rt\sys\darwin\osevent.d \
-	$(IMPDIR)\rt\sys\darwin\osmutex.d \
+	$(IMPDIR)\rt\sys\config.d \
+	\
+	$(IMPDIR)\rt\sys\darwin\config.d \
 	$(IMPDIR)\rt\sys\darwin\ossemaphore.d \
-	$(IMPDIR)\rt\sys\darwin\oscondition.d \
 	\
-	$(IMPDIR)\rt\sys\freebsd\osevent.d \
-	$(IMPDIR)\rt\sys\freebsd\osmutex.d \
```

Comment:
why are so many of these files removed?

Reply:
Because it is a change of strategy how to include the POSIX implementation. Previously the files that are removed just do a public import of the generic POSIX implementation file.

Now instead this is set in the config.d file for each OS. For example. ```enum string osMutexImport = "rt.sys.posix.osmutex";```

tells the implementation of the Mutex to include OsMutex from the POSIX implementation directly using ```mixin("import " ~ osMutexImport ~ ";");``` instead.

Reply:
ah ok i see. might as well hit merge in a few minutes then if you're happy with this stuff

Reply:
Much of the other OS API stuff is based on this and it has worked somewhat well I'm good with it. There might better ways but in that case they have to write a proposal.

Reply:
ok then rebase the others on master and fix merge conflicts and they can go too